pulp_container/tests/functional/api/test_rbac_repo_content.py

```diff
@@ -182,3 +183,85 @@ def test_rbac_repository_content(
         container_bindings.ContentTagsApi.read(pushed_tag.pulp_href)
     with user_helpless, pytest.raises(container_bindings.ApiException):
         container_bindings.ContentTagsApi.read(pushed_tag.pulp_href)
+
+
+def test_rbac_label_content(
+    gen_user,
+    container_bindings,
+    container_remote_factory,
+    container_repository_factory,
+    container_repository_version_api,
+    container_sync,
+    pulp_settings,
+):
+
+    def _do_test(the_binding, the_content, rv):
+        # Set label
+        sl = SetLabel(key="key_1", value="value_1")
+        the_binding.set_label(the_content.pulp_href, sl)
+        the_content = the_binding.read(the_content.pulp_href)
+        labels = the_content.pulp_labels
+        assert "key_1" in labels
+
+        # Search for key_1
+        rslt = the_binding.list(pulp_label_select="key_1", repository_version=rv.pulp_href)
+        assert 1 == rslt.count
+
+        # Change an existing label
+        sl = SetLabel(key="key_1", value="XXX")
+        the_binding.set_label(the_content.pulp_href, sl)
+        the_content = the_binding.read(the_content.pulp_href)
+        labels = the_content.pulp_labels
+        assert labels["key_1"] == "XXX"
+
+        # Unset a label
+        sl = UnsetLabel(key="key_1")
+        the_binding.unset_label(the_content.pulp_href, sl)
+        content2 = the_binding.read(the_content.pulp_href)
+        assert "key_1" not in content2.pulp_labels
+
+    if pulp_settings.TOKEN_AUTH_DISABLED:
+        pytest.skip("RBAC cannot be tested when token authentication is disabled")
```

Comment:
```suggestion
```
This RBAC check is only for the Registry API. Since this is the Pulp API we should be good.

Reply:
@ggainey You removed the wrong lines. That is why the s3 test is failing now.

Reply:
....ha!  Code in haste, repent at leisure :)